### ISSUE TITLE
Announce this publicly once it is ready for wider usage

### ISSUE THREAD

**Issue by certik** [lfortran/lcompilers_frontend#34]
Once we fix the following issues:

* [ ] CI (#15)
* [ ] Runtime library (#10)

And generally fix the most basic bugs (let's update the above list with everything that needs fixing), we should:

* [ ] Add a link to lfortran.org
* [ ] Add a link to fortran-lang (https://github.com/fortran-lang/webpage/issues/46)
* [ ] Write a blog post

**Comment by Shaikh-Ubaid** [comment]
- It seems we have a basic CI which builds the site and uploads it to https://github.com/lfortran/pull_request_preview. (It currently overrides the previous deployment as shared here https://github.com/lfortran/lcompilers_frontend/issues/15#issuecomment-1242991611). This preliminary CI could help ensure if the page can build and if the site works.
- We also have basic support for the `runtime` library at https://dev.lfortran.org/ (Functions like kind(), abs(), floor(), etc. currently work at https://dev.lfortran.org/)

**Comment by certik** [comment]
Yes, for the runtime library the remaining issue to fix are the parts implemented in C. However, the current version is already very usable, so we should announce it and add it to lfortran.org.
